operator/pom.xml

```diff
@@ -155,6 +155,19 @@
               <generateMarkdown>true</generateMarkdown>
             </configuration>
           </execution>
+          <execution>
+            <id>generate-cluster-schema</id>
+            <phase>process-classes</phase>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+            <configuration>
+              <rootClass>oracle.kubernetes.weblogic.domain.model.ClusterResource</rootClass>
+              <outputFile>oracle/kubernetes/weblogic/domain/model/Cluster.json</outputFile>
+              <kubernetesVersion>1.13.5</kubernetesVersion>
```

Comment:
This can be a todo as well, but it seems odd that we still have content and a file with a Kubernetes version of "1.13.5". That seems wrong.